### PR TITLE
Email Validation

### PR DESCRIPTION
**Pull request by emyrue**
Used javascript to verify that the email entered was all lowercase. Displayed an error message otherwise.

### CODE REVIEW

File: javascript.js [hunk 1]
```diff
@@ -100,4 +100,28 @@ for (let m = 0; m < seeProject.length; m += 1) {
...
+function validateEmail(input) {
+  const address = input.toString();
+  let isLower = false;
+  if (address === address.toLowerCase()) {
+    isLower = true;
+  } else {
+    isLower = false;
+  }
+  return isLower;
+}
```
Comment (keroloussamy) [inline javascript.js:118]: - [ ] Kindly when I enter an email without capital letters it shows me the error message. Kindly fix it.

image

File: index.html [hunk 2]
```diff
@@ -270,7 +270,8 @@ <h2 class="contact-heading">Get started</h2>
                 ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
                 quis nostrud exercitation ullamco laboris nisi ut aliquip ex 
                 ea commodo consequat. </p>
-            <form class="form" action="https://formspree.io/f/mjvlwyao" method="POST">
+            <small></small>
+            <form class="form" id="form" action="https://formspree.io/f/mjvlwyao" method="POST">
```
Comment (keroloussamy) [inline index.html:274]: - [ ] Kindly make the error message near the submit button as required in the requirements.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @emyrue, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by medaminedev66 — APPROVED** [review]
## Hi @emyrue , 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**